src/aiidalab_qe/app/result/components/viewer/structure/structure.py

```diff
@@ -26,11 +27,22 @@ def _render(self):
             """)
             self.atom_coordinates_table = TableWidget()
             self._generate_table(structure.get_ase())
-            self.results_container.children = [
+
+            # Basic widgets
+            children = [
                 self.widget,
                 self.table_description,
                 self.atom_coordinates_table,
             ]
+
+            # Add structure info if it is a relaxed structure
+            if "relax" in self._model.properties:
+                self._initialize_structure_info(structure)
+                children.insert(0, self.structure_info)
+
+            # Add the children to the container
+            self.results_container.children = tuple(children)
+
```

Comment:
```suggestion

            children = []

            if "relax" in self._model.properties:
                structure_info = self._get_structure_info(structure)
                children.append(structure_info)

            children.extend(
                [
                    self.widget,
                    self.table_description,
                    self.atom_coordinates_table,
                ]
            )

            self.results_container.children = children

```

Also, should this only be for relaxed structures? Why not for any? If you agree, remove the check.

Reply:
Because for the initial structure, we have the info in the summary ? if you want i can add if is it not relaxed

Reply:
I think in both is good? I'm about to meet with Giovanni. I'll let you know what he says.

Reply:
lets do any